README.md

```diff
@@ -305,73 +437,157 @@ Once _all_ the previous steps have been performed
   **NOT READY FOR REVIEW** in the description if it's still in progress.
 
-### B) _Prepare_ a Pull Request (PR) for Review
+### B) _Prepare_ a Pull Request (PR) and _Assign_ (_to someone else_) for Review
 
 > **Rule (of thumb)**: _**Keep Pull Requests Small and Focussed**_.
 
-1. Make sure ***all tests*** are **passing**
+1. ***Describe*** what your PR contains in the PR description;
+it only takes a _few seconds_
+but ***dramatically improves*** the ***review process***. <br />
+(_Please **never** leave the PR description blank!
+  remember this is in an exercise in **communicating with your team members**_!)
+
++ Make sure ***all tests*** are **passing**
   and (_where applicable_) that your **code coverage** is ***100%***.
   + ***Never*** _comment_ out a test that is failing.
   (_Don't laugh, we have seen "**cowboy developers**" do this...!
   If a test is failing, the feature is broken!
   Fix the feature to make the test pass. Only then is your work "**done**"_)
   + ***Never*** _assign_ a PR for "_review_"
   _until_ _**all CI checks** are **pass**_.
+  ![check failed](https://cloud.githubusercontent.com/assets/194400/22284176/db2d698a-e2dd-11e6-8799-573226328b6b.png) <br />
+  if _anything_ is failing on a PR and you _need_ help simply
+  ***comment*** in the PR "_thread_" _asking_ for help!
 
-+ **Increment** the _version_ of your application if it is a "_new release_".
++  **Increment** the _version_ of your application if it is a "_new release_".
 
 + ***Remove*** the `[WiP]` from your PR title.
 
-+ ***Describe*** what your PR contains in the PR description. <br />
-(_**Never** leave the PR description blank!
-  remember this is in an exercise in **communicating with your team members**_!)
-
 + **Remove** the `in-progress` label from the PR and associated issues.
 
-+ ***Apply*** the `awaiting-review` label to the PR.
-
-+ ***Assign*** someone to review your PR.
++ ***Apply*** the
+[`awaiting-review` **label**](https://github.com/dwyl/labels/issues/49) to the PR.
+
++ ***Assign*** someone `else` to _review_ your PR. <br />
+(_please don't merge your own PRs we need to ensure every change
+has been reviewed by at least one other person on the team. thanks!_)
+
+### C) _Reviewer(s): Review_, _Comment On_ and _Merge_ the Pull Request
+
+Always remember: a **Pull Requests** is a
+"_**discussion thread** for **understanding** the code_",
+our objective is to _ensure_ the code works, is well documented/tested
+and is "_free from errors_". We don't
+["_nit pick_"](https://en.oxforddictionaries.com/definition/nitpicking)
+if it's going to delay shipping. Equally, where code requires _clarification_,
+tell people; ***everyone*** benefits from easy-to-read (_understand_) code.
+
+The person who is _assigned_ to review the PR
+(_often referred to as the **Quality Assurance Person** or "**QA**"_) will:
+
+1. ***Confirm*** that all "**CI**" ***checks pass*** _before_ doing anything else.
```

Comment:
I think it's in _both_ places. perhaps we need to reduce it down to just the part _before_ the PR gets assigned. 👍